src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -493,19 +528,19 @@ def notify_sentry_app(event: GroupEvent, futures: Sequence[RuleFuture]):
 
 def send_webhooks(installation: RpcSentryAppInstallation, event: str, **kwargs: Any) -> None:
     servicehook: ServiceHook
+    extras: dict[str, int | str] = {"installation_id": installation.id, "event": event}
     try:
         servicehook = ServiceHook.objects.get(
             organization_id=installation.organization_id, actor_id=installation.id
         )
     except ServiceHook.DoesNotExist:
-        logger.info(
-            "send_webhooks.missing_servicehook",
-            extra={"installation_id": installation.id, "event": event},
+        raise SentryAppSentryError(
```

Comment:
do we need to make sure the callers of this `send_webhooks` function are correctly retrying/not retrying on this exception?

Reply:
we ignore do not retry on `SentryAppSentryErrors` for these tasks via the retry decorator. If there's logic specific to a caller we can add them when we get to their SLO